config_defaults_inc.php

```diff
@@ -1837,14 +1837,25 @@
 #############################################
 
 /**
- * Login authentication method. Must be one of
- * MD5, LDAP, BASIC_AUTH or HTTP_AUTH.
+ * Login authentication method.
+ *
+ * The default SAFE_HASH method relies on PHP's password_hash() function, using
+ * the default algorithm (PASSWORD_DEFAULT, currently CRYPT_BLOWFISH).
+ * @see http://php.net/function.password-hash
+ *
  * Note: you may not be able to easily switch encryption methods, so this
- * should be carefully chosen at install time. However, MantisBT will attempt
- * to "fall back" to older methods if possible.
- * @global integer $g_login_method
- */
-$g_login_method = MD5;
+ * should be carefully chosen at install time. However, if possible, MantisBT
+ * will attempt to "fall back" to older methods, and to convert the DB-stored
+ * hashes to the current login method after a successful login (e.g. with
+ * default settings, a MD5 hash will be upgraded to SAFE_HASH).
+ *
+ * @global integer $g_login_method One of: SAFE_HASH (default), LDAP, BASIC_AUTH
+ *                                 or HTTP_AUTH. Other values (PLAIN, MD5, CRYPT
```

Comment:
The Admin Checks will issue a warning in this case (including for MD5 now), I think that's sufficient - let the administrator decide.

We could obsolete them completely if you want to but I think that's outside of this PR's scope.